### ISSUE TITLE
vk3d install error

### ISSUE THREAD

**Issue by gazhay** [Heroic-Games-Launcher/HeroicGamesLauncher#1347]
### Describe the bug

Linux AppImage, Heroic was working fine, prompted for update, now on launch I get the following error 

````
 Extraction of vkd3d failed with: Error: Command failed: tar -I zstd -xvf /home/gaz/.config/heroic/tools/vkd3d/vkd3d-proton-2.6.tar.zst -C /home/gaz/.config/heroic/tools/vkd3d
````

Blank window

### Add logs

Cannot access UI as it never loads

### Steps to reproduce

Launch heroic appimage

### Expected behavior

Launch, or error message displayed.

### System Information

- OS :  Ubuntu 20.04LTS 
- Heroic Version : 2.3.2

**Comment by gazhay** [comment]
installing zstd gets you past the error above, but UI still does not load

**Comment by flavioislima** [comment]
where is the rest of the log?

**Comment by gazhay** [comment]
````
QApplication: invalid style override passed, ignoring it.
    Available styles: Windows, Fusion
(16:15:46) DEBUG:   [Legendary]:       Running Legendary command: /tmp/.mount_Heroicjz2Du5/resources/app.asar.unpacked/build/bin/linux/legendary --version
[839800:0520/161546.998030:ERROR:gpu_memory_buffer_support_x11.cc(44)] dri3 extension not supported.
(16:15:47) INFO:    [Legendary]:       Legendary location: /tmp/.mount_Heroicjz2Du5/resources/app.asar.unpacked/build/bin/linux/legendary
(16:15:47) INFO:    [Gog]:             GOGDL location: /tmp/.mount_Heroicjz2Du5/resources/app.asar.unpacked/build/bin/linux/gogdl
(16:15:47) INFO:    [Backend]:         
  Heroic Version: 2.3.2 Brook
  Legendary Version:  0.20.25 Our Benefactors
  OS: Ubuntu KERNEL: 5.4.0-110-generic ARCH: x64
  CPU: AMD Ryzen 9 3950X 16-Core Processor @3.5 GOVERNOR: ondemand
  RAM: Total: 94.31 GiB Available: 85.87 GiB
  GRAPHICS: GPU0: NVIDIA GeForce RTX 3070 Ti VRAM: 7979MB DRIVER: 470.129.06 
  PROTOCOL: x11
  
Checking for update
(16:15:47) ERROR:   [Backend]:         Failed to register protocol with OS.
[839762:0520/161547.656077:ERROR:browser_main_loop.cc(269)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[839762:0520/161547.656148:ERROR:browser_main_loop.cc(269)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[839762:0520/161547.656197:ERROR:browser_main_loop.cc(269)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
Update for version 2.3.2 is not available (latest version: 2.3.2, downgrade is disallowed).

````
